projects/admin/src/app/app.module.ts

```diff
@@ -106,21 +118,17 @@ import { PersonDetailViewComponent } from './record/detail-view/person-detail-vi
 import { VendorDetailViewComponent } from './record/detail-view/vendor-detail-view/vendor-detail-view.component';
 import { ItemAvailabilityComponent } from './record/item-availability/item-availability.component';
 import { AppConfigService } from './service/app-config.service';
-import { SharedPipesModule } from './shared/shared-pipes.module';
-import { MarcPipe } from './pipe/marc.pipe';
-import { TabOrderDirective } from './directives/tab-order.directive';
-import { ItemsBriefViewComponent } from './record/brief-view/items-brief-view/items-brief-view.component';
 import { UiRemoteTypeaheadService } from './service/ui-remote-typeahead.service';
-import { RemoteTypeaheadService } from '@rero/ng-core';
-import { MefTypeahead } from './class/mef-typeahead';
-import { DocumentsTypeahead } from './class/documents-typeahead';
-import { MainTitlePipe } from './pipe/main-title.pipe';
 import { AppInitService } from './service/app-init.service';
 
 /** Init application factory */
 export function appInitFactory(appInitService: AppInitService) {
   return () => appInitService.load();
 }
+import { SharedPipesModule } from './shared/shared-pipes.module';
+import {
+  HoldingItemInCollectionComponent
+} from './record/detail-view/document-detail-view/holding/holding-item-in-collection/holding-item-in-collection.component';
```

Comment:
Why these imports are not placed with others ?

Reply:
done